ur_robot_driver/doc/usage_example.md

```diff
@@ -0,0 +1,204 @@
+teach panel and watch how it also moves inside ROS.
+
+![RViz window](resources/usage_example/rviz.png "RViz window showing a UR5 robot")
+
+## Control the robot
+For controlling the robot you'll need a special program running on the robot that will interpret and
+execute the commands coming from ROS. This can be either done by installing the **External Control**
+URCap as explained
+[here](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/README.md#setting-up-a-ur-robot-for-ur_robot_driver)
+or by using the [headless
+mode](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/README.md#headless-mode).
+We assume the use of the URCap in the following.
+
+Before any control of the robot can happen, a program containing the **External Control** program
+node must be running on the robot. You should see the output `Robot ready to receive control
+commands.` in the terminal where you launched the driver. See [this troubleshooting
+entry](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/README.md#i-started-everything-but-i-cannot-control-the-robot)
+if you experience problems in this step.
+
+### Control the robot using the `test_move` script
+Once you've setup the robot for getting controlled by ROS, you can move it around using the
+[`test_move`](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/scripts/test_move) script.
+
+Start the driver and an RViz window as before. Then, in a third terminal run
+
+```
+rosrun ur_robot_driver tes_move
+```
+
+This script will guide you through the different trajectory controllers available. For informations
+about the different controllers, please see [TODO: Change once merged #423](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/423).
+
+It will first ask about the controller to be used:
+
+```
+Available trajectory controllers:
+0 (joint-based): scaled_pos_joint_traj_controller
+1 (joint-based): scaled_vel_joint_traj_controller
+2 (joint-based): pos_joint_traj_controller
+3 (joint-based): vel_joint_traj_controller
+4 (joint-based): forward_joint_traj_controller
+5 (Cartesian): pose_based_cartesian_traj_controller
+6 (Cartesian): joint_based_cartesian_traj_controller
+7 (Cartesian): forward_cartesian_traj_controller
+Please choose a controller by entering its number (Enter '0' if you are unsure / don't care): 0
+```
+
+After that, you'll have to confirm that the motions the robot is about to make, can safely be
+executed the way your robot is mounted. If you're running a simulated robot, you can confirm this
+without thinking twice. However, if a real robot is connected, please make sure that the robot is
+actually free of any obstacles inside the space where the motion is taking place. See the images
+below for a visualization of the setpoints:
+
+**Joint-based motions**: (The cable leaving the base to the back on an e-Series robot)
```

Comment:
When looking at this in the rendered .md file, it seems a little unorganized (one on top and two below). Might also be my current screen width interfering with this. As an alternative, I would propose using a single image for this (and also the Cartesian illustrations). This could be done with either using [RViz' trail mechanism](https://answers.ros.org/question/199552/rviz-robotmodel-base_link-show-trail-history/), or using an overlay of partly transparent images. If the screenshots are taken from the same camera angle, the second should be possible.

Reply:
I've created an overlayed image of all: 
![image](https://user-images.githubusercontent.com/491645/130206477-ec84205e-86f4-4491-8116-8748996b6369.png)

It looks a bit confusing, but it very clearly shows the configurations being used inside the trajectory, which is what we want in that case. @stefanscherzinger do you think, this might be the way to go?

Reply:
A little dark, but I like it! Let's use that.

Reply:
Resolved in 426ea83